docs/ssm-on-aws/ssm-on-aws.md

```diff
@@ -0,0 +1,374 @@
+
+[Adrian Hornsby][AdrianHornsbyMedium] recently wrote an excellent article on [Injecting Chaos to Amazon EC2 using AWS System Manager][InjectingChaosToAmazonEc2]. This highlighted the fact that this was not covered in the ChaosIQ documentation. The article itself does a great job of covering what AWS Systems Manager (SSM) is and how to set it up.
```

Comment:
This paragraph might have to be cleaned up a bit.

Reply:
Yeah, I would remove the bit about our docs because it's *now* out of date, surely ;)